### ISSUE TITLE
Refine AMA and Disability content (continued)

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#3247]
**Note: this is associated with a Feb. 19 launch**

This is a continuation of ticket [2609](url https://github.com/department-of-veterans-affairs/va.gov-team/issues/2609) - expanded scope due to additional SME feedback. 

Also related is ticket [5541](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5541).

**The pages are:**
1/ File a VA disability appeal   http://preview-prod.vfs.va.gov/preview?nodeId=901
2/ VA decisions reviews and appeals https://vagov-content-pr-570.herokuapp.com/decision-reviews/
3/ After you request a decision review https://vagov-content-pr-570.herokuapp.com/decision-reviews/after-you-request-review/
4/ File a Supplemental Claim with new evidence  https://vagov-content-pr-570.herokuapp.com/decision-reviews/supplemental-claim/
5/ Request a Higher-Level Review  https://vagov-content-pr-570.herokuapp.com/decision-reviews/higher-level-review/
6/ Request a Board Appeal  https://vagov-content-pr-570.herokuapp.com/decision-reviews/board-appeal/

**Fred Somers additional feedback document:**
[VAGovRevisions-decision-reviews-appeals-11.04.19-fredsomers.docx](https://app.zenhub.com/files/133843125/84a716d8-07e9-4e52-973f-c622eb1a9336/download)

[AMA Changes for Feb 19 2020 (FS 1.6.19).doc](https://app.zenhub.com/files/133843125/f14a9b23-7d21-49de-af37-46a1a2008429/download)  <-- MORE CHANGES FOR FEB 19 PER FRED'S 1/7/20 EMAIL

## Acceptance Criteria
- [ ]

**Comment by Karak888** [comment]
@lwalsh617  just tagging you for this ticket per our standup this morning.

**Comment by jenniferlee-dsva** [comment]
@lwalsh617 - Assigning you this. And will assign the other one that just came in from Fred Somers regarding the forms issues. Just heads up - not ready yet.

**Comment by jenniferlee-dsva** [comment]
@lwalsh617 @Karak888 - Added Fred's change doc for Feb 19 updates. 

- Laura please take a look after your HLR work and recommend if those Feb 19 updates can be incorporated into this 'refinements' ticket of if it should be kept separate. 
- Kara: Feb 19 content updates have a Feb 19 (TBC) go live deadline -- currently confirming w/Fred if this needs to be day-of (which would mean everything has to be ready the day before) or the week of, which would give us a little wiggle room to go live the week of. Could you please add the appropriate sprint milestone?

**Comment by lwalsh617** [comment]
@jenniferlee-dsva -- Here's the document I referenced in Slack. This doc contains both rounds of Fred's feedback. I indicated those areas where I'm proposing copy edits with SUGGESTED REWORK in red, all caps. It's only on pages 1-4 and 20-22. 
I know you can't review until next week. Would it be easiest to do a call? I can walk you through edits/rationale and get your feedback? If so DM me and I'll shoot you my availability. Thanks!
[VA.gov LA and DR pages_content Qs and suggestions.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4066552/VA.gov.LA.and.DR.pages_content.Qs.and.suggestions.docx)

**Comment by MickinSahni** [comment]
Preparing with call with stakeholders Friday. 
Laura does need help with Drupal pages from @RLHecht, will send artifacts along. 
Need to remove banner site on Decision Review page --  the alert banner. Will need @kelsonic, @ncksllvn to remove this. @lwalsh617 to spin up a separate ticket to track.

**Comment by RLHecht** [comment]
@lwalsh617 All the content changes have been made to the File a VA Disability Appeals page. There seems to be an issue with preview in Drupal where it's not showing the text for the callout numbers we moved to the bottom of the page. @kevwalsh suggested that I publish on staging and you can review with stakeholders from there. Let me know what you want me to do. 
<img width="894" alt="Screen Shot 2020-02-06 at 10 13 21 AM" src="https://user-images.githubusercontent.com/48728214/73950086-57fc4b80-48c9-11ea-85f9-2b3a1cc1d160.png">
<img width="1098" alt="Screen Shot 2020-02-06 at 10 13 55 AM" src="https://user-images.githubusercontent.com/48728214/73950145-6a768500-48c9-11ea-8a08-8609bc25a461.png">

**Comment by MickinSahni** [comment]
2020-02-06 PW Standup
- next: 
   - develop doc to share with stakeholders then review - Laura
     - requires mtg with Mikki + Nick before doc is finalized + FE/graphic changes 
   - make one more change - Randi

**Comment by RLHecht** [comment]
@lwalsh617 The link for "Manage a legacy appeal" was added to the Disability hub landing page. 

I also changed the H1 and meta title tag for the About VA legacy appeals per your Slack. Drupal won't let me change the menu title ("VA legacy appeals") until the page is published though so it's still going to show "File an appeal." We need to make a note to change it before it goes live. 

This should complete all the changes you needed me to make for this ticket but let me know if there's anything outstanding.

**Comment by MickinSahni** [comment]
2020-02-07 PW Standup
- Randi's portion is done
- Open PR for decision review updates
- @lwalsh617 can you confirm you have all you need?

**Comment by lwalsh617** [comment]
@MickinSahni - I have what I need for today. SME feedback is due Feb 13, which we want to incorporate in time to send updated Drupal links, Heroku links and screenshots back to SMEs by 2/18.  I'll make all the GH edits I can, but will need help from Nick and the cms team on just a couple of pieces, as well as Randi's help with final Drupal edits.  (I expect this next round of edits to be minimal as the updates aren't extensive.)

**Comment by MickinSahni** [comment]
Thanks, @lwalsh617! Couple questions:

- Can we close this ticket out? 
- If we close this out, will you need a new ticket to track the next round?

**Comment by MickinSahni** [comment]
Preemptively pulling into sprint 15 so that we can get these changes made and deployed on 2/19.

**Comment by MickinSahni** [comment]
2020-02-13 PW Standup
- @lwalsh617 hey laura, when you have a moment, can you update the acceptance criteria for this particular ticket? 
we _believe_ any changes that randi needs to make is already captured in #5541 but if there's content that needs to be changed we can track that here.

**Comment by MickinSahni** [comment]
2020-02-19 PW Standup
- making final updates and will share links with jen (and stakeholders)
- anticipating wrapping up by eod

**Comment by MickinSahni** [comment]
hey @lwalsh617 checking in to see if we're still gtg with wrapping this up by end of sprint

**Comment by jenniferlee-dsva** [comment]
@MickinSahni - I am reviewing Laura's latest updates and have changes for her. As she is out today for family situation, I am making these updates directly in CMS and will pull in @RLHecht for certain nav links assistance. We should be able to send these out for SME final review Friday, but we should give them a day to provide concurrence - so likely not until Monday or Tuesday for go live.

**Comment by AkucherGCIO** [comment]
AMA Launch ticket - https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/6165

Closing this ticket as we are launching.
